how-to/secure/enable-tls-clients.md

```diff
@@ -94,9 +113,17 @@ If the `ssl-ca` parameter is not specified in the TiDB server or MySQL client, t
 
 - To perform mutual authentication, meet both of the above requirements.
 
+By default, you can choose to authenticate the client from the server. Even if the client does not present its certificate of identification during the TLS handshake, the TLS connection can be established. You can also require the client to be authenticated through `require 509` when creating a user (`create user`), granting permissions (`grant`), or modifying an existing user (`alter user`). The following is an example of creating a user:
+
+{{< copyable "sql" >}}
+
+```sql
+create user 'u1'@'%'  require x509;
+```
+
 > **Note:**
 >
-> Currently, it is optional that TiDB server authenticates the client. If the client does not present its identity certificate in the TLS handshake, the TLS connection can also be successfully established.
+> If the logged-in user has been configured to verify the user certificate using the [TiDB Certificate-Based Authentication for Login] (/reference/security/cert-based-authentication.md#configure-the-user-certificate-information-for-login-verification), the user is implicitly required to enable the encrypted connection to TiDB.
```

Comment:
@lysu Please confirm whether `如果登录用户已配置使用 TiDB 证书鉴权功能校验用户证书` means `If the logged-in user has configured using the TiDB Certificate-Based Authentication for Login`, thanks!

Reply:
maybe we should remove "ed" from "logged-in user"? yes， you means we need "如果登录用户已配置使用 TiDB 证书鉴权功能校验用户证书" instead of "如果已登录用户已配置使用 TiDB 证书鉴权功能校验用户证书"？

Reply:
@lysu I will update `logged-in user later`. But Tom wants to confirm whether `如果登录用户已配置使用 TiDB 证书鉴权功能校验用户证书` means `If the logged-in user has configured using the TiDB Certificate-Based Authentication for Login`. Here the Chinese version is slightly different from the English version~ The English version omits `校验用户证书`, since `校验用户证书` is implicit in the sentence

Reply:
emmm~ I agree with you, we can omits "校验用户证书" :D